rootfs/conf/kops/helmfile.yaml

```diff
@@ -1146,6 +1207,13 @@ releases:
     - name: "backends.docs.external"
       value: "true"
 
+    - name: "backends.kibana.name"
```

Comment:
Because ElasticSearch is $$ I think we should not add it by default. Maybe add a conditional for `PORTAL_BACKEND_KIBANA_ENABLED`